bridged-nft-token/src/internal.rs

```diff
@@ -0,0 +1,167 @@
+use crate::*;
+use near_sdk::{log, CryptoHash};
+use std::mem::size_of;
+
+pub(crate) fn hash_account_id(account_id: &AccountId) -> CryptoHash {
+    let mut hash = CryptoHash::default();
+    hash.copy_from_slice(&env::sha256(account_id.as_bytes()));
+    hash
+}
+
+pub(crate) fn assert_one_yocto() {
+    assert_eq!(
+        env::attached_deposit(),
+        1,
+        "Requires attached deposit of exactly 1 yoctoNEAR",
+    )
+}
```

Comment:
Use from near_sdk rather than reimplementing?

Reply:
it was removed and replaced by the one contained inside the standard